backend/tests/test_auth_api.py

```
@@ -2,160 +1,70 @@
+#             "address": "Somewhere",
+#             "phone": "0123456789",
+#             "role": "customer"
+#         },
+#     )
+#     assert response.status_code == 200
+#     data = response.json()
+#     assert "access_token" in data
+#     assert data["token_type"] == "bearer"
+
+
+def test_login_user(override_get_db, test_db):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_login_user`

Add return type annotation: `None`

(ANN201)